www/src/pages/docs/index.js

```diff
@@ -25,18 +25,16 @@ class IndexRoute extends React.Component {
             <p>There are four main ways to get started with Gatsby:</p>
             <ol>
               <li>
-                <Link to="/tutorial/">Tutorial</Link>: The tutorial is written
-                to be as accessible as possible to people without much web
-                development experience.
+                <Link to="/tutorial/">Tutorial</Link>: Go to the tutorial for step-by-step instructions on how to install Gatsby and start a project. Great for people new to Gatsby and is written to be accessible to people without web
```

Comment:
Trying to condense this a bit:
```suggestion
                <Link to="/tutorial/">Tutorial</Link>: Step-by-step instructions on how to install Gatsby and start a project: written for people without Gatsby or web development experience, though it has helped developers of all skill levels.
```